modules/responses.js

```diff
@@ -0,0 +1,47 @@
+
+
+exports.sendError = function(res) {
+	var error = {
+		status: 0,
+		message: "Error in execution"
+	}
+	res.send(error);
+}
+
+exports.success = function(result, res) {
+	var response = {
```

Comment:
everytime the response should be same

Reply:
I will implement the status codes in this code . I did not learn about the status codes till now.